docs: update artifactory wh docs for self-hosted users (#5103)
akuity/kargo: docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/artifactory/index.md

new artifacts from all repositories to which that `Warehouse` subscribes.
:::

## Self-Hosted Artifactory

:::info
If you are not using a self-hosted Artifactory instance, skip to
[the configuring the receiver](#configuring-the-receiver) section.
:::

In order for a webhook initiated `Warehouse` refresh to successfully occur,
it is required that you set a <Hlt>Custom Base URL</Hlt> for your instance. 
When this setting hasn't been configured, critical information will be missing 
from the webhook payloads.

1. Navigate to 

> `https://<jfrog-instance>.jfrog.io/ui/admin/configuration/general`, where `<jfrog-instance>` has been replaced with an Artifactory instance for which you are an administrator.

Reviewer: ```suggestion
`https://<base-url>/ui/admin/configuration/general`, where `<base-url>` has been replaced with the base URL of your self-hosted Artifactory instance.
```

> 1.  Set the <Hlt>Custom Base URL</Hlt> field to your instance's URL.

Reviewer: ```suggestion
1.  Set the <Hlt>Custom Base URL</Hlt> field to the base URL of your self-hosted
    Artifactory instance.
```

new artifacts from all repositories to which that `Warehouse` subscribes.
:::

## Special note for self-hosted Artifactory users

:::info
If you are not using a self-hosted Artifactory instance, skip to
[the configuring the receiver](#configuring-the-receiver) section.
:::

In order for a webhook initiated `Warehouse` refresh to successfully occur,

> it is required that you set a <Hlt>Custom Base URL</Hlt> for your instance.

Reviewer: It may be worthwhile to elaborate on this (only a little) by explaining that when this setting hasn't been configured, critical information will be missing from the webhook payloads.

1. Click on the <Hlt>Administration</Hlt> tab at the top.

    ![Admin Tab](./img/admin_tab.png "Admin Tab")

1. Hover over <Hlt>General Management</Hlt> in the left-hand sidebar.

1. Click <Hlt>Settings</Hlt>.

    ![General Mgmt](./img/general_management.png "General Mgmt")

1.  Set the <Hlt>Custom Base URL</Hlt> field to your instance's URL.

    ![Custom Base URL](./img/custom_base_url.png "Custom Base URL")

> 1. Scroll all the way down to the bottom and click <Hlt>Save</Hlt>.

Reviewer: Nit: This wording is just landing awkwardly. May I suggest:

```suggestion
1. At the bottom of the form, click <Hlt>Save</Hlt>.
```

> 
> 

Reviewer: ```suggestion
```
